pkg/doc/signature/verifier/public_key_verifier.go

```diff
@@ -272,6 +273,18 @@ func (sv *ECDSASignatureVerifier) Verify(pubKey *PublicKey, msg, signature []byt
 	r := big.NewInt(0).SetBytes(signature[:ec.keySize])
 	s := big.NewInt(0).SetBytes(signature[ec.keySize:])
 
+	if len(signature) > 2*ec.keySize {
```

Comment:
it would be good to have unit test for this case

Reply:
I will add it later